main.cpp

```diff
@@ -133,256 +143,329 @@ struct CarWash
 
 struct Computer
 {
-    // amount of storage
     float storageAmount = 30.31f;
-    // amount of RAM
     int RAMAmount = 16;
-    // manufacturer name
     std::string manufacturer = "Mac";
-    // size of screen
     float screenSize = 13.1f;
-    // number of usb ports (int)
     int amountUSBPort = 2;
 
-    // 3 things it can do:
-    // 1) connect to internet
-    void connectToInternet();
-    // 2) save files
-    void save();
-    //     3) run apps
-    void runApp();
+    bool connectToInternet(bool paidISP = true);
+    bool save(std::string fileName, bool clickSaveButton);
+    bool runApp(bool doubleClick);
 };
 
+bool connectToInternet(bool paidISP = true)
+{
+    return paidISP ? true : false;
+}
+
+bool save(std::string fileName, bool clickSaveButton)
+{
+    return fileName != "" && clickSaveButton ? true : false;
+}
+
+bool runApp(bool doubleClick)
+{
+    return doubleClick ? true : false;
+}
```

Comment:
These are not member functions.  these are free functions.
You did not fully qualify the function name
- [ ] revise